Tests/UserServiceTests/Commands/CreateUserCommandTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using NUnit.Framework;
+using UserService.Interfaces;
+using UserService.Commands;
+using FluentValidation;
+using DTO.RestRequests;
+using Moq;
+using MassTransit;
+using DTO.BrokerRequests;
+using DTO;
+using System.Threading.Tasks;
+using FluentValidation.Results;
+using Kernel.CustomExceptions;
+
+namespace UserServiceTests
+{
+   public class CreateUserCommandTests
```

Comment:
Move it to the right a bit

Reply:
> Move it to the right a bit


These three lines?

Reply:
> > Move it to the right a bit
> 
> These three lines?

`public class CreateUserCommandTests` should be moved a bit to the right, does not code like

```
namespace N
{
     class X
         {
         }
}
```
bother you?

Reply:
> > > Move it to the right a bit
> > 
> > 
> > These three lines?
> 
> `public class CreateUserCommandTests` should be moved a bit to the right, does not code like
> 
> ```
> namespace N
> {
>      class X
>          {
>          }
> }
> ```
> 
> bother you?


Fixed